pkg/langruntime/langruntime.go

```diff
@@ -65,6 +74,13 @@ func (l *Langruntimes) ReadConfigMap() {
 			logrus.Fatalf("Unable to get the runtime images: %v", err)
 		}
 	}
+
+	if provisionImages, ok := l.kubelessConfig.Data["provision-container-image"]; ok {
```

Comment:
I am not convinced about this since this package was meant to be to handle language specific logic. It is a bit weird that now `CronJobs` depends on this package even though it is language specific. I would move this out of this package. We could create a standalone package with the new interface, instantiate it in the Controller `New` method and pass it to the `EnsureFuncDeployment` and `EnsureCronJob`.

We will need to do that separation anyway when we split the controllers in different entities.